llvm/test/Transforms/InstCombine/phi-int-users.ll

```diff
@@ -0,0 +1,576 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -mtriple=arm64 -passes='inline,function(sroa<modify-cfg>,jump-threading,instcombine)' -S < %s | FileCheck %s
```

Comment:
Why can't this be an instcombine only test?

Reply:
True, changed